xdsl/pattern_rewriter.py

```diff
@@ -86,60 +86,47 @@ class PatternRewriter(PatternRewriterListener):
     has_done_action: bool = field(default=False, init=False)
     """Has the rewriter done any action during the current match."""
 
+    def insert_op_at_location(
+        self, op: Operation | Sequence[Operation], insertion_point: InsertPoint
+    ):
+        """Insert operations at a certain location in a block."""
+        self.has_done_action = True
+        op = (op,) if isinstance(op, Operation) else op
+        if not op:
+            return
+        for op_ in op:
+            Rewriter.insert_op_at_location(op_, insertion_point)
```

Comment:
Actually, `Rewriter.insert_op_at_location` should allow a sequence of operations? Or is that not the case?